ArduPlane/quadplane.cpp

```diff
@@ -1385,7 +1386,8 @@ void QuadPlane::control_loiter()
         relax_attitude_control();
         pos_control->relax_alt_hold_controllers(0);
         loiter_nav->clear_pilot_desired_acceleration();
-        loiter_nav->init_target();
+        bool init_I_terms = !_is_vectored;
```

Comment:
we should move _is_vectored into the tailsiller structure, so it becomes init_I_terms = tailsitter.is_vectored.
It is quite misleading as it is, as we have multiple types of vectored aircraft, only one of which is a tailsitter

Reply:
good idea... done